features/patroni_api.feature

```diff
@@ -13,13 +13,34 @@ Scenario: check API requests on a stand-alone server
 	When I issue an empty POST request to http://127.0.0.1:8008/reinitialize
 	Then I receive a response code 503
 	And I receive a response text "I am the leader, can not reinitialize"
-	When I issue a POST request to http://127.0.0.1:8008/failover with leader=postgres0
+	When I issue a POST request to http://127.0.0.1:8008/failover with {"leader": "postgres0"}
 	Then I receive a response code 500
-	And I receive a response text "failover is not possible: cluster does not have members except leader"
+	And I receive a response text failover is not possible: cluster does not have members except leader
 	When I issue an empty POST request to http://127.0.0.1:8008/failover
 	Then I receive a response code 400
+	When I issue a POST request to http://127.0.0.1:8008/failover with {"foo": "bar"}
+	Then I receive a response code 400
 	And I receive a response text "No values given for required parameters leader and candidate"
 
+Scenario: check local configuration reload
+        Given I issue an empty POST request to http://127.0.0.1:8008/reload
+        Then I receive a response code 304
+        When I add tag new_tag new_value to postgres0 config
+        And I issue an empty POST request to http://127.0.0.1:8008/reload
+        Then I receive a response code 202
+
+Scenario: check dynamic configuration change via DCS
+        Given I issue a PATCH request to http://127.0.0.1:8008/config with {"ttl": 20, "loop_wait": 1, "postgresql": {"parameters": {"max_connections": 101}}}
+        Then I receive a response code 200
+        And I receive a response loop_wait 1
+        And Response on GET http://127.0.0.1:8008/patroni contains restart_pending after 11 seconds
```

Comment:
Shall we rename to pending_restart, we'll be compatible with 
9.5+ then:
https://www.postgresql.org/docs/9.5/static/view-pg-settings.html